test/remote/gateways/remote_trans_first_transaction_express_test.rb

```diff
@@ -109,6 +109,21 @@ def test_successful_purchase_with_empty_string_cvv
     assert_equal 'Succeeded', response.message
   end
 
+  def test_successful_purchase_without_name
+    credit_card_opts = {
+      :number => 4485896261017708,
+      :month => Date.new((Time.now.year + 1), 9, 30).month,
+      :year => Date.new((Time.now.year + 1), 9, 30).year,
+      :first_name => '',
+      :last_name => ''
+    }
+
+    credit_card = CreditCard.new(credit_card_opts)
+    response = @gateway.purchase(@amount, credit_card, @options)
+    assert_success response
+    assert_equal 'Succeeded', response.message
```

Comment:
Is it worth looking for the name being absent from the request? Or will the sandbox error if you send it blank

Reply:
It could be a unit test, alternatively.

Reply:
I did manually test that, but probably a good idea.

Reply:
Added case to same test.

Reply:
I guess I was thinking about verifying that the blank name field is not added to the request. But I wasn't sure if you needed to write that into the test necessarily, or if the sandbox would error in that case.

Reply:
Ah, I see. Yes, this was in response to the gateway throwing an error when the field was sent as an empty string, so the remote test should be inherently sufficient.